cts/scheduler/op-defaults.exp

```diff
@@ -0,0 +1,74 @@
+<transition_graph cluster-delay="60s" stonith-timeout="60s" failed-stop-offset="INFINITY" failed-start-offset="INFINITY"  transition_id="0">
+  <synapse id="0">
+    <action_set>
+      <rsc_op id="5" operation="monitor" operation_key="ip-rsc_monitor_0" on_node="cluster02" on_node_uuid="2">
+        <primitive id="ip-rsc" class="ocf" provider="heartbeat" type="IPaddr2"/>
+        <attributes CRM_meta_on_node="cluster02" CRM_meta_on_node_uuid="2" CRM_meta_op_target_rc="7" CRM_meta_timeout="7000" cidr_netmask="32"  ip="172.17.1.1"/>
+      </rsc_op>
```

Comment:
All of the scheduled actions are 0-second monitors, so they all match op-ping-defaults and get a 7s timeout

None of the resources are being started b/c there's no fence device. If you add one, you should get starts and recurring monitors scheduled, which will let you test the other matches

Reply:
I added a fence device and verified all the timeouts match what you've described here, which was also what I intended them to be.  I additionally renamed the ping-related op_default name to op-monitor-defaults, since that's what it does.